### DISCUSSION TITLE
Using Impact Framework to export data into CSV

### DISCUSSION THREAD

**Discussion by tdigeser91** [Green-Software-Foundation/discussions#177] [Ask Anything] [answered]
How can I use the if-csv to export more than one parameter to the export csv file? And do you have more documentation on this function?

**Comment by NAMRATA-WOKE** [comment]
@jmcook1186 can you help answer this question?

**Reply by jmcook1186** [reply, marked as answer]
We've discussed this over on the IF issues board.
